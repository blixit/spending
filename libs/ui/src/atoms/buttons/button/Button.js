import React from 'react';
import styled from 'styled-components';

const StyledButton = styled.button`
  min-height: 30px;
  lineheight: 25px;
  padding: 5px;
  cursor: pointer;
`;

const Button = props => <StyledButton {...props} />;

export default Button;
