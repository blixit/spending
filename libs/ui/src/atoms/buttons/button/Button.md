```jsx
  <Button>My button</Button>
```