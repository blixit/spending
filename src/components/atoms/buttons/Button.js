import React from 'react';
import styled from 'styled-components';

const StyledButton = styled.button`
  min-height: 25px;
  lineheight: 25px;
  font-size: 16px;
  padding: 5px;
  cursor: pointer;
`;

class Button extends React.Component {
  render() {
    return (
      <StyledButton {...(this.props)} />
    );
  }
}

export default Button;
