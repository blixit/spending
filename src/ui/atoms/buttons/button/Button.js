import React from 'react';
import styled from 'styled-components';

const StyledButton = styled.button`
  min-height: 24px;
  lineheight: 25px;
  padding: 5px;
  cursor: pointer;
`;

const Button = props => <StyledButton {...props} />;

Button.displayName = 'Button';

export default Button;
