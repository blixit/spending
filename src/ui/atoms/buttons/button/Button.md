```jsx
<ExampleWrap inline >
  <Button>My button</Button>
  <Button>My button</Button>
</ExampleWrap>
```