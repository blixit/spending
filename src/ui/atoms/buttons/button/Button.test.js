import React from 'react';

import render from 'testing/render';

import Button from './Button';

describe('[atoms/buttons/button] Button', () => {
  it('should render properly', () => {
    const tree = render(<Button />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
