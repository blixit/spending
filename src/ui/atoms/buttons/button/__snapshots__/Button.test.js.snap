// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[atoms/buttons/button] Button should render properly 1`] = `
<button
  className="sc-bdVaJa cbeqcr"
/>
`;
